### ISSUE TITLE
Insufficient Contrast Ratio for 'Service Updates' heading

### ISSUE THREAD

**Issue by sgroi-l** [localgovdrupal/localgov_base#606]
### Description

The black 'Service Updates' h2 title on a purple background does not meet the minimum contrast ratio requirements. The contrast ratio is currently 2.18.

### Expected Behaviour

- Regular text (14pt or 18.5px and below) should have a contrast ratio of at least 4.5:1 with its background.
- Large-scale text (14pt bold, 18pt or 24px and above) should have a contrast ratio of at least 3:1 with its background.

### Actual Behaviour

The contrast ratio for the Service Updates' h2 title on the purple background is 2.18.

![image](https://github.com/user-attachments/assets/0d7f30f8-2872-45bc-a187-3e8550fdf1b6)

### How to Replicate

1. Navigate to a service page. eg https://a11y.localgovdrupal.org/adult-health-and-social-care
2. Observe the title against the purple background.
3. Use a contrast checker tool to verify that the contrast ratio is 2.18.

### WCAG Reference

- **WCAG Guideline 1.4.3: Contrast (Minimum) (AA)**  
  *The visual presentation of regular text (14pt or 18.5px and below) and images of text has a contrast ratio of at least 4.5:1. Large-scale text and images of large-scale text (14pt bold, 18pt or 24px and above) have a contrast ratio of at least 3:1.*

More details: [WCAG 1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

### Recommendations

1. Adjust the colour of the h2 or the purple background to ensure the contrast ratio meets the minimum requirement.

**Comment by millnut** [comment]
This should be fixed by https://github.com/localgovdrupal/localgov_base/pull/604
